app/models/rating.rb

```diff
@@ -0,0 +1,7 @@
+class Rating < ApplicationRecord
+  belongs_to :user
+  belongs_to :movie
+
+  validates :body, length: { in: 5..255 }
+  validates :rating, numericality: { only_integer: true }, inclusion: { in: 1..5 }
```

Comment:
it's up to you, but maybe it makes sense to make the rating 10 points? tell me what you think about it
![image](https://user-images.githubusercontent.com/70864999/155401782-673e07f3-3f09-40ad-9593-77fd6edfac56.png)
10 point scale is used everywhere in the  kinopoisk

Reply:
yeah you are right, that's looks better